<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
        <link rel="stylesheet" href="./static/css/theme.css">
        <script src="./static/js/jquery-3.2.1.min.js"></script>
        <script src="./static/js/bootstrap.min.js"></script>
    <title>转录组结题报告</title>
</head>



<body>
<div class="container-fluid">
    <h3><b>公司已发表论文</b></h3>
    <p>
    上海嘉因生物科技有限公司，是致力于高通量实验设计，高通量实验实施及生物信息学分析的医学科研服务企业。我们的核心成员来自于哈佛大学、加州大学洛杉矶分校、清华大学和同济大学等国内外知名学府。截至目前，团队成员已在Nature，Cancer Cell，Genome Research，Nature Structural & MolecularBiology，Leukemia 和 Nucleic Acids Research等国际顶尖学术期刊发表20余篇学术论文，累计影响因子超过220。<br>
<br>
1)The modENCODE Consortium. modENCODE and ENCODE resources for analysis of metazoan chromatin organization. Nature, 2014. (IF:42.4)<br>
2)The JARID1B/KDM5B histone demethylase is a luminal lineage-driving oncogene in breast cancer. Cancer Cell, 2014. (IF:23.9)<br>
3)Canonical nucleosome organization at promoters forms during genome activation. Genome Research, 2014. (IF: 13.9)<br>
4)Integrative genomic analyses reveal clinically relevant long non-coding RNA in human cancer. Nature Structural & Molecular Biology, 2013. (IF:11.6)<br>
5)Integrating Gene and Mir Expression Profiles and Regulatory Network Structures to Define Aberrent Feed Forward Loops with Functional and Clinical Implications in Myeloma. Blood, 2012. (IF:9.8)<br>
6)Transcription factor-pathway coexpression analysis reveals cooperation between SP1 and ESR1 on dysregulating cell cycle arrest in non-hyperdiploid multiple myeloma. Leukemia, 2013. (IF:9.7)<br>
7)CR Cistrome: a ChIP-Seq database for chromatin regulators and histone modification linkages in human and mouse. Nucleic Acids Research, 2013. (IF: 8.8)<br>
8)PlantGSEA: a gene set enrichment analysis toolkit for plant community. Nucleic Acids Research, 2013. (IF: 8.8)<br>
9)agriGO: a GO analysis toolkit for the agricultural community. Nucleic Acids Research, 2010. (IF: 8.8) <br>
10)Target analysis by integration of transcriptome and ChIP-seq data with BETA. Nature Protocol, 2013. (IF: 7.8)<br>
11)Identifying ChIP-seq enrichment using MACS. Nature Protocol, 2012.(IF: 7.8)<br>
12)Genome-wide analysis of histone modifications: H3K4me2, H3K4me3, H3K9ac, and H3K27ac in Oryza sativa L. Japonica. Molecular Plant, 2013. (IF:6.6)<br>
13)BSeQC: Quality Control of Bisulfite Sequencing Experiments. Bioinformatics, 2013. (IF: 5.3)<br>
14)CistromeFinder for ChIP-seq and DNase-seq data reuse. Bioinformatics. 2013. (IF: 5.3)<br>
15)DiNuP: a systematic approach to identify regions of differential nucleosome positioning. Bioinformatics, 2012. (IF: 5.3)<br>
16)GFOLD: a generalized fold change for ranking differentially ex-pressed genes from RNA-seq data. Bioinformatics, 2012. (IF: 5.3)<br>
17)CistromeMap: A knowledgebase and web server for ChIP-Seq and DNase-Seq studies in mouse and human. Bioinformatics, 2012. (IF: 5.3)<br>
18)PHF8 and REST/NRSF co-occupy gene promoters to regulate proximal gene expression. Scientific report, 2014. (IF:5.1)<br>
19)Local chromatin dynamics of transcription factors imply cell lineage-specific functions during cellular differentiation. Epigenetics, 2012. (IF: 5.1)<br>
20)GeSICA: genome segmentation from intra-chromosomal associations. BMC Genomics, 2012. (IF: 4.0)<br>
21)ProFITS of maize: a database of protein families involved in the transduction of signaling in the maize genome. BMC genomics, 2010. (IF: 4.0)<br>
22)plantsUPS: a database of plants’ Ubiquitin Proteasome System. BMC genomics, 2009. (IF: 4.0)<br>
23)Classify hyperdiploidy status of multiple myeloma patients using gene expression profiles. PLoS ONE,2013. (IF:3.5)<br>

</p>
</div>

</body>
</html>