<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
        <link rel="stylesheet" href="./static/css/theme.css">
        <script src="./static/js/jquery-3.2.1.min.js"></script>
        <script src="./static/js/bootstrap.min.js"></script>
    <title>转录组结题报告</title>
</head>



<body>
<div class="container-fluid">
    <h2><b>Reference</b></h2>
    <p>
    [1] Andrews S. FastQC: A quality control tool for high throughput sequence data[J]. Reference Source, 2010.<br>
    [2] Martin M. Cutadapt removes adapter sequences from high-throughput sequencing reads[J]. EMBnet. journal, 2011, 17(1): pp. 10-12.<br>
    [3] Dobin A, Davis C A, Schlesinger F, et al. STAR: ultrafast universal RNA-seq aligner[J]. Bioinformatics, 2013, 29(1): 15-21.<br>
    [4] Shen S, Park J W, Lu Z, et al. rMATS: robust and flexible detection of differential alternative splicing from replicate RNA-Seq data[J]. Proceedings of the National Academy of Sciences, 2014, 111(51): E5593-E5601.<br>
    [5] Anders S, Pyl P T, Huber W. HTSeq–a Python framework to work with high-throughput sequencing data[J]. Bioinformatics, 2014: btu638.<br>
    [6] Love M I, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-Seq data with DESeq2. bioRxiv, 2014[J]. Genome Biology, 2014, 10: 002832.<br>

    </p>
</div>

</body>
</html>